sensor/debugger/k8s/k8sclient.go

```diff
@@ -88,6 +90,12 @@ func (c *ClientSet) OpenshiftRoute() routeVersioned.Interface {
 	return c.openshiftRoute
 }
 
+// OpenshiftOperator returns the OpenshiftOperator interface
+// This is not used in tests!
```

Comment:
what is used then? (I know this is just copied from others, but I'm not familiar with this part of the codebase)

Reply:
Is in order to adhere to the `client.Interface` interface in `sensor/kubernetes/client/client.go`.  Looks like it is used when creating fake clients for various tests.